Dockerfile

```diff
@@ -1,61 +1,59 @@
-FROM phusion/baseimage:0.9.15
+FROM phusion/baseimage:0.9.18
 MAINTAINER Open Knowledge
 
-# Disable SSH
-RUN rm -rf /etc/service/sshd /etc/my_init.d/00_regen_ssh_host_keys.sh
-
-ENV HOME /root
+# Environment
 ENV CKAN_HOME /usr/lib/ckan/default
 ENV CKAN_CONFIG /etc/ckan/default
 ENV CKAN_DATA /var/lib/ckan
+ENV CKAN_GIT_BRANCH release-v2.5.2
```

Comment:
Develop against master and when we fork for 2.6 it should just work for the release too.